packages/components/src/__actions__/Button/v3/_docs/Button.stories.tsx

```diff
@@ -19,80 +21,53 @@ export default meta
 type Story = StoryObj<typeof meta>
 
 export const Playground: Story = {
```

Comment:
No need for the render :)
```suggestion
export const Playground: Story = {}
```